File: src/easy/_1816_截断句子.java
package easy;

public class _1816_截断句子 {
    public String truncateSentence(String s, int k) {
        int len = s.length();
        int end = 0, count = 0;
        for (int i = 1; i <= len; i++) {
            if (i == len || s.charAt(i) == ' ') {
                count++;
                if (count == k) {
                    end = i;
                    break;
                }
            }
        }
        return s.substring(0, end);
    }
}

File: src/go/easy/1816_截断句子.go
package easy

func truncateSentence(s string, k int) string {
	len := len(s)
	end := 0
	count := 0
	for i := 0; i <= len; i++ {
		if i == len || s[i] == ' ' {
			count++
			if k == count {
				break
			}
		}
	}
	return s[:end]
}
